### PR TITLE
76 Installer check automation should load hashes from yaml file

### LINKED ISSUE #76: Installer check automation should load hashes from yaml file

### PR DESCRIPTION
**Pull request by kylemccullen**
Closes #76

### CODE REVIEW

File: Branta/Branta.csproj [hunk 1]
```diff
@@ -64,6 +68,7 @@
   <ItemGroup>
     <PackageReference Include="Countly" Version="24.1.0" />
     <PackageReference Include="Hardcodet.NotifyIcon.Wpf" Version="1.1.0" />
+    <PackageReference Include="YamlDotNet" Version="15.1.2" />
```
Comment (keith-gardner) [inline Branta/Branta.csproj:71]: 👍 

CVE's look clear: https://www.cvedetails.com/vulnerability-list/vendor_id-19028/product_id-48676/Yamldotnet-Project-Yamldotnet.html

Later on we can add a CVE to github actions or CI, like https://snyk.io/blog/snyk-code-security-scanning-c-sharp-dot-net/

File: Branta/Automation/Wallets/BaseWallet.cs [hunk 2]
```diff
@@ -18,8 +18,6 @@ public abstract partial class BaseWallet
 
     public abstract IReadOnlyDictionary<string, string> CheckSums { get; }
 
-    public abstract IReadOnlyDictionary<string, string> InstallerHashes { get; }
```
Comment (keith-gardner) [inline Branta/Automation/Wallets/BaseWallet.cs:21]: LGTM.

Do you see any downsides to this change? I don't think we lose anything of value.

File: Branta/Assets/InstallerHashes.yaml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# BlockstreamGreen. Standard SHA 256 from manifest.
```
Comment (keith-gardner) [inline Branta/Assets/InstallerHashes.yaml:1]: `InstallerHashes` seems like a good enough name, but open to suggestions

File: Branta/Automation/Wallets/BaseWallet.cs (hunk 2 above)
Comment (kylemccullen) [reply Branta/Automation/Wallets/BaseWallet.cs:21]: I don't see any downsides once we have up to date hashes in the yaml file.

**Review by keith-gardner — APPROVED** [review]

**Pull request merged**